evennia/scripts/manager.py

```diff
@@ -85,7 +85,7 @@ def get_all_scripts(self, key=None):
             script = []
             dbref = self.dbref(key)
             if dbref or dbref == 0:
-                script = [self.dbref_search(dbref)]
+                script = filter(None, [self.dbref_search(dbref)])
```

Comment:
I know this is closed, but I just noticed, should this be *self.filter* and not just filter or where is filter coming from? I don't see it defined or in imports.